guides/source/developers/promotions/overview.html.md

```diff
@@ -140,7 +138,7 @@ See the `eligible?` method defined in the [Spree::Promotion
 model][spree-promotion]:
 
 ```ruby
-# models/spree/promotion.rb : line 123
+# models/spree/promotion.rb : line 120
```

Comment:
I'm not sure it's a good idea to keep this line number. It's very hard to maintain and it's super easy to find a method into a file with all editors or browsers. Not sure if it's better to take advantage of this change to remove it or we could remove it also in other places (if any) with another PR. WDYT?

Reply:
Updated Thanks for the suggestion 😄